WordPress/WordPressShareExtension/ShareModularViewController.swift

```diff
@@ -234,6 +234,24 @@ extension ShareModularViewController {
         clearSiteDataAndRefreshSitesTable()
         reloadSitesIfNeeded()
     }
+
+    func showTagsPicker() {
+        guard let siteID = shareData.selectedSiteID, isPublishingPost == false else {
+            return
+        }
+
+        let tagsPicker = ShareTagsPickerViewController(siteID: siteID, tags: shareData.tags)
+        tagsPicker.onValueChanged = { tagString in
```

Comment:
I think we might need `[weak self]` here? And then you can probably just optionally chain `(self?.refreshModulesTable()`) all the references to `self` below.